sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ConnectionPolicy.java

```diff
@@ -497,6 +500,14 @@ public ConnectionPolicy setMaxRequestsPerConnection(int maxRequestsPerConnection
         return this;
     }
 
+    public boolean isClientTelemetryEnabled() {
+        return clientTelemetryEnabled;
+    }
+
+    public void setClientTelemetryEnabled(boolean clientTelemetryEnabled) {
+        this.clientTelemetryEnabled = clientTelemetryEnabled;
+    }
+
     @Override
     public String toString() {
         return "ConnectionPolicy{" +
```

Comment:
Please include `clientTelemetryEnabled` in `toString()` API

Reply:
Good catch , thanks